docs/lib/auth/fragments/android/hub_events/10_listen_events.md

```diff
@@ -32,17 +32,15 @@ Amplify.Hub.subscribe(HubChannel.AUTH,
  <amplify-block name="Kotlin">
 
  ```kotlin
-Amplify.Hub.subscribe(HubChannel.AUTH) { hubEvent: HubEvent<*> ->
-    if (hubEvent.name == InitializationStatus.SUCCEEDED.toString()) {
+Amplify.Hub.subscribe(HubChannel.AUTH).collect {
```

Comment:
style/nit: when expressions tend to be more readable than multi-conditional if blocks

Reply:
Updating -- thanks!